hal/src/core/pinmap_impl.h

```diff
@@ -36,6 +36,7 @@ extern "C" {
 typedef struct STM32_Pin_Info {
   GPIO_TypeDef* gpio_peripheral;
   pin_t gpio_pin;
+  uint8_t gpio_pin_source;
```

Comment:
Totally, I only noticed afterwards. I should have written a follow up note -  my appologies for the noise!